src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -13,7 +11,7 @@ namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
     /// <summary>
     /// Abstract class used to buffer content returned by <see cref="ITagHelper"/>s.
     /// </summary>
-    public abstract class TagHelperContent : IEnumerable<string>, IHtmlContent
+    public abstract class TagHelperContent : IHtmlContent
```

Comment:
A `TagHelperContent` being a list of strings seems odd to me, given that we have a new type that represents this concept better. No? HTML content is more complex than simply being a list of strings...